src/h5web/vis-packs/nexus/pack-utils.test.ts

```diff
@@ -103,6 +108,19 @@ describe('getSupportedVis', () => {
     expect(getSupportedVis(unknown)).toBe(NEXUS_VIS[NexusVis.NxImage]);
   });
 
+  it('should fall back when dataset is incompatible with interpretation', () => {
```

Comment:
fall back to what ?

Reply:
Same as the test above (the dataset shape), not sure it was worth repeating as it lengthens an already long test name ^^

Reply:
Yeah but the sentence feels weird... I would prefer it to be repeated ^^

Reply:
Fair enough 👍